File: 2-Module/1-week/3-day/managerAndEmployeeProject/employee.js
class Employee{
  constructor(name, salary, title, manager = null){
    this.name = name;
    this.salary = salary;
    this.title = title;
    this.manager = manager
    if (manager) manager.addEmployee(this);
  }

  calculateBonus(multiplier){
    let bonus = this.salary * multiplier;
    return bonus
  }

}

module.exports = Employee;

File: 6-Module/2-week/4-day/employee_manager/employee.py
class Employee:
    def __init__(self, name, salary, title, manager=None):
        self.name = name
        self.salary = salary
        self.title = title
        self.manager = manager
        if manager:
            manager.add_employee(self)

    def calculate_bonus(self, multiplier):
        return self.salary * multiplier
